ipaserver/install/cainstance.py

```diff
@@ -1506,6 +1510,73 @@ def uncomment_rewriterule():
                 logger.debug("Successfully updated CRL")
             api.Backend.ra.override_port = None
 
+    def setup_acme(self) -> bool:
+        """
+        Set up ACME service, if needed.
+
+        Return False if ACME service was already set up, otherwise True.
+
+        """
+
+        # ACME LDAP database schema will be added by ipa-server-upgrade.
+        # It is fine if this subroutine runs *before* the schema update,
+        # because we only create the container objects.
+
+        if os.path.isdir(os.path.join(paths.PKI_TOMCAT, 'acme')):
+            return False  # it seems ACME service is already set up
```

Comment:
Perhaps a logging.debug() since you are catching this?

Reply:
Fixed in next push.